### ISSUE TITLE
Evaluation on kinetics 600

### ISSUE THREAD

**Issue by gilgoldm** [Atze00/MoViNet-pytorch#44]
Hi,

I am trying to reproduce the MoViNet-A0-Stream model on the Kinetics test set, specifically on the part of the test annotations that are publicly available here: [https://storage.googleapis.com/deepmind-media/Datasets/kinetics600.tar.gz](https://storage.googleapis.com/deepmind-media/Datasets/kinetics600.tar.gz)

I get 69%, which is 3% less than the results published on the TensorFlow [repository](https://github.com/tensorflow/models/tree/master/official/projects/movinet).

It may not be an issue with the translation of the code from TF to PyTorch. I was wondering if anyone managed to reproduce the 72% result.
